tests/test_pcap_eager.py

```diff
@@ -34,8 +32,19 @@ def test_pcap_input():
   pcap_filename = os.path.join(
       os.path.dirname(os.path.abspath(__file__)), "test_pcap", "http.pcap")
   file_url = "file://" + pcap_filename
-  url_filenames = [file_url]
-  dataset = pcap_io.PcapDataset(url_filenames, batch=1)
+
+  dataset = tfio.IODataset.from_pcap(file_url, capacity=5).batch(1)
+
+  packets_total = 0
+  for v in dataset:
+    (packet_timestamp, packet_data) = v
+    if packets_total == 0:
+      assert np.isclose(packet_timestamp.numpy()[0], 1084443427.311224) # we know this is the correct value in the test pcap file
```

Comment:
since it's a big number you might want to add the `rtol` param to something like `rtol=1e-15`

Reply:
@BryanCutler Updated. Thanks.